packages/perseus/src/index.ts

```diff
@@ -190,7 +190,8 @@ export type {
     Size,
     CollinearTuple,
     MathFormat,
-    InputNumberWidget, // TODO(jeremy): remove?
+    InputNumberWidget, // Used for usurpation of InputNumberWidget in perseus-editor
+    NumericInputWidget, // Used for usurpation of InputNumberWidget in perseus-editor
     // Widget configuration types
```

Comment:
Note to self: I should be able to fix this if I use the PerseusWidgetMap type -- then we don't need to export these.